main.js

```diff
@@ -0,0 +1,148 @@
+        origin: 'Ireland',
+        imageUrl: 'https://goo.gl/YNb74O'
+      },
+      {
+        itemId: 8,
+        name: 'TR-09',
+        brand: 'Roland',
+        price: 399.00,
+        description: 'A drum machine for people with no desk space.',
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Japan',
+        imageUrl: 'https://goo.gl/gJYUxz'
+      }
+    ]
+  },
+  details: {
+    item: null
+  }
+}
+
+var $app = document.getElementById('app')
+var $h1 = document.querySelector('h1')
+
+function displayItem(itemNumber) {
+
+  var $card = document.createElement('div')
+  var $img = document.createElement('img')
+  var $cardBody = document.createElement('div')
+  var $cardTitle = document.createElement('h5')
+  var $cardSubTitle = document.createElement('h6')
+  var $cardText = document.createElement('p')
+  var $link = document.createElement('a')
+
+  $card.setAttribute('class', 'card')
+  $card.setAttribute('style', 'width: 18rem;')
+  $img.setAttribute('class', 'card-img-top')
+  $img.setAttribute('src', 'src=.../1--px180/')
+  $img.setAttribute('alt', 'Card image cap')
+  $cardBody.setAttribute('class', 'card-body')
+  $cardTitle.setAttribute('class', 'card-title')
+  $cardSubTitle.setAttribute('class', 'card-subtitle mb-2 text-muted')
+  $cardText.setAttribute('class', 'card-text')
+  $link.setAttribute('href', '#')
+  $link.setAttribute('class', 'btn btn-outline-primary btn-sm')
+
+  $cardTitle.textContent = app.catalog.items[itemNumber].name
+  $cardSubTitle.textContent = app.catalog.items[itemNumber].brand
+  $img.src = app.catalog.items[itemNumber].imageUrl
+  $cardText.textContent = app.catalog.items[itemNumber].price
+  $link.textContent = 'Add to cart'
+
+  $card.appendChild($img)
+  $card.appendChild($cardTitle)
+  $card.appendChild($cardSubTitle)
+  $card.appendChild($cardBody)
+  $card.appendChild($cardText)
+  $card.appendChild($link)
+
+  return $card
+}
+
+function buildCatalog() {
+  for (var i = 0; i < app.catalog.items.length; i++) {
+    var item = displayItem(i)
+    $app.appendChild(item)
+  }
+  $app.style.visibility = 'hidden'
+  $h1.textContent = 'Jamazon'
+  $h1.style.visibility = 'hidden'
+}
+
+function renderCatalog() {
```

Comment:
`renderCatalog` should be named something like `renderApp` and it should take the whole `app` as its argument.
Then get the items from the catalog and build the `$catalog` with your "pure" `buildCatalog` function. At that point you can append the built catalog to the `data-view="catalog"` in `index.html`.